homeassistant/components/refoss/strings.json

```diff
@@ -0,0 +1,13 @@
+{
+  "config": {
+    "step": {
+      "confirm": {
+        "description": "[%key:common::config_flow::description::confirm_setup%]"
+      }
+    },
+    "abort": {
+      "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%]",
+      "no_devices_found": "[%key:common::config_flow::abort::no_devices_found%]"
+    }
+  }
```

Comment:
You should also add the other strings that could happen when failing, I think there are more than this.

Reply:
Currently, there is nothing else here, and Gree only has these。